src/standard/gestures.html

```diff
@@ -78,6 +86,23 @@
       Polymer.Debounce(POINTERSTATE.mouse.mouseIgnoreJob, unset, MOUSE_TIMEOUT);
   }
 
+  function hasLeftMouseButton(ev) {
+    var type = ev.type;
+    if (MOUSE_EVENTS.indexOf(type) === -1) {
+      return false;
+    }
+    if (type === 'mousemove') {
+      var buttons = ev.buttons === undefined ? 1 : ev.buttons;
```

Comment:
Given the relative obscurity of the values for `buttons` and other similar properties, it might be nice to riddle this whole function with comments explaining the significance of the checks and the corresponding default values.